### ISSUE TITLE
Closing, I used wrong template (TrimMultilineRawString support for alternate trimming methods)

### ISSUE THREAD

**Issue by samuelneff** [detekt/detekt#5414]
The `TrimMultilineRawString` (https://github.com/detekt/detekt/pull/5051) rule can be helpfully extended to allow custom trimming functions and chained functions. 

## Expected Behavior
Here is an example from our codebase:

```kotlin
fun String.collapseWhitespace(text: String) =
    trim().replace(Regex("\\s\\s+"), " ")

val blockWithoutAnyExtraWhitespace = """
    This is some text.

    Here is another line; spaced apart.
""".collapseWhitespace()

println(blockWithoutAnyExtraWhitespace)
// Prints: This is some text. Here is another line; spaced apart.
```

## Current Behavior

Currently the rule requires `trimIndent()` or `trimMargin()` be called and nothing else is considered valid.

## Context

See above example from our codebase that fails this rule but we would like to be able to pass. 
Configuration with this change would include an additional option:

```yml
TrimMultilineRawString:
  additionalTrimmingMethods:
    - collapseWhitespace
```

Note: I'm working on another rule now for Hacktoberfest. If I actually get one done, I'll look into implementing this as well. Looks straightforward. Thanks for this great new rule!

**Comment by samuelneff** [comment]
I used the wrong template. See replacement issue https://github.com/detekt/detekt/issues/5415.

Sorry.
